src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
-// write code here
+const buttonAppendRow = document.querySelector('.append-row');
+
+buttonAppendRow.addEventListener('click', () => {
+  const tbody = document.querySelector('tbody');
+
+  if (tbody.children.length < 10) {
```

Comment:
you should also disable/enable buttons depending on row/columns count:
![image](https://user-images.githubusercontent.com/86923089/204040246-f7a413f0-d6d2-4344-87b8-c4685b9bf7fc.png)